Client/Frontend/TabContentsScripts/DownloadContentScript.swift

```diff
@@ -56,12 +56,7 @@ class DownloadContentScript: TabContentScript {
               let data = Bytes.decodeBase64(base64String)
         else { return }
 
-        // TODO: Could we have a download queue per tab instead of resolving from foregroundBVC?
-        // Or one that is independent of BVC at least?
-        let browserViewController = BrowserViewController.foregroundBVC()
-
         defer {
-            browserViewController?.pendingDownloadWebView = nil
```

Comment:
How come this isn't needed?

Reply:
I believe `pendingDownloadWebView` gets reset every time in [BVC+WebViewDelegates](https://github.com/mozilla-mobile/firefox-ios/blob/4d4876781e880a582deab565db901d2ce1cf19eb/Client/Frontend/Browser/BrowserViewController/BrowserViewController%2BWebViewDelegates.swift#L673).

Reply:
Do you know how to trigger this code path? I couldn't hit it by just downloading a random file.

Reply:
I tried a lot but can't really find a way to trigger it. But I made sure for each download, the `pendingDownloadWebView` does get `nil`ed out prior to the next one starting.

Reply:
@nbhasin2 Do you know how to trigger this code path? I tried a bunch of different download types from both clicking links and the long press menu and couldn't trigger it. 

I'm not super comfortable with this change if we don't understand what it's doing. If it is just a dead code path tho then we should create a follow up task to remove it altogether.

Reply:
Let me try a flow that might work and revert back here

Reply:
This is for `blob` downloads and should not be removed unless you are replacing this with something equally in line with the change.

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/8919439/221287974-d9785697-2730-4642-9853-5820298d5c37.png">

<img width="1099" alt="image" src="https://user-images.githubusercontent.com/8919439/221288172-063cfdc6-c6d5-4f4a-9918-a576817940ff.png">

Reply:
You can try it by long pressing on a valid URL `blob:https://....`
Simple google should be able to get you some download URL websites to test blob download

Reply:
Gotcha - I'll undo that particular change.